roles/slave_tools/tasks/nodejs.yml

```diff
@@ -132,19 +132,52 @@
     PATH: /opt/build/tools/node-v8.11.3-linux-x64/bin:{{ansible_env.PATH}}
   tags: nodejs
 
+- name: Check for Node 10.12
+  stat: path=/opt/build/tools/node-v10.12.0-linux-x64
+  register: nodelts_installed
+- name: Get Node 10.12 package
+  s3: bucket={{s3_bucket}} region={{s3_region}}
+      object=/{{s3_tools_path}}/node-v10.12.0-linux-x64.tar.xz dest=/tmp/node-v10.12.0-linux-x64.tar.xz mode=get
+      aws_access_key={{aws_id.msg}} aws_secret_key={{aws_secret.msg}}
+  when: not nodelts_installed.stat.exists
+- name: Install Node 10.12
+  unarchive: src=/tmp/node-v10.12.0-linux-x64.tar.xz dest=/opt/build/tools copy=no owner=root group=root
+  when: not nodelts_installed.stat.exists
+- name: Install npm 6.4.1
+  npm: name=npm version=6.4.1 global=yes
+  environment:
+    PATH: /opt/build/tools/node-v10.12.0-linux-x64/bin:{{ansible_env.PATH}}
+- name: Install Node 10.12 dependencies
+  npm: name={{item.name}} version={{item.version}} global=yes
+  with_items:
+  - name: yo
+    version: 2.0.5
+  - name: grunt-cli
+    version: 1.3.1
+  - name: gulp-cl1
+    version: 2.0.1
+  - name: bower
+    version: 1.8.4
+  - name: unicode
+    version: 11.0.1
```

Comment:
Do we need unicode as a globally installed package? It doesn't have an CLI API.

Reply:
This is one of those "if it ain't broken don't fix it" situations.. tbh I just kept same set of global packages